packages/fxa-auth-server/lib/inactive-accounts/index.ts

```diff
@@ -216,9 +226,110 @@ export class InactiveAccountsManager {
       return;
     }
 
+    const account = await this.fxaDb.account(taskPayload.uid);
+    const emails = account.emails.map((e) => e.email);
+    const now = Date.now();
+
+    // If the very first email hard bounced, we'll delete the account without
+    // sending further emails.
```

Comment:
👍🏽

Reply:
nit: This might be helpul to be pulled into its own function for readiability.

Reply:
Would we need to wrap this block in a try/catch? ie if the account delete task fails?

Reply:
This in response to a cloud task so the task would retry.